README.md

```diff
@@ -24,29 +24,101 @@ There are two modules, the datasources and the pipeline:
 
 ![Indicadors_schema](/docs/Indicadors.jpg "Schema")
 
-## install
+## Install using `virtualenvwrapper`
 
-```
+The following commands require `virtualenvwrapper`.
+
+```sh
 mkvirtualenv dades
 pip install -r requirements.txt
 cp dbconfig.example.py dbconfig.py
 ```
 
+## Install using `pyenv`, `pipx` and `poetry`
+
+The following commands require `pyenv`, `pipx` and `poetry`.
+
+### Quickstart
+
+Assuming `poetry` and `pyenv` are already installed, you can install the dependencies with the following commands:
+
+```sh
+pyenv install 3.8.12
+pyenv virtualenv 3.8.12 somenergia-kpis
+pyenv activate somenergia-kpis
+poetry install
+```
+
+Read the following sections for more details.
+
+### Using `pyenv`
+
+1. Install `pyenv` using [pyenv-installer](https://github.com/pyenv/pyenv-installer) and follow their instructions. Pay special attention to the following steps:
+   1. Extending your `.bashrc` file, documented in [the pyenv-installer README](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv), so that `pyenv` is available in your shell.
+   2. Install `pyenv` dependencies for installing python versions as per the [pyenv wiki](https://github.com/pyenv/pyenv#install-python-build-dependencies)
+2. Install the required python version with `pyenv install 3.8.12`, and wait until it finishes.
+3. Create a new virtual environment of your choice. If you used `pyenv-installer` you should be able to create one with `pyenv virtualenv 3.8.12 somenergia-kpis`. Here, `somenergia-kpis` is the name of the virtual environment.
+4. Activate the virtual environment with `pyenv activate somenergia-kpis`. Alternatively, you can use `pyenv shell somenergia-kpis` to activate the virtual environment for the current shell session. You can also set the python version locally with `pyenv local somenergia-kpis`. This will create a file called `.python-version` which is used by pyenv to automatically activate the virtual environment when you `cd` into the directory.
+
+### Using `pipx`
+
+`pipx` acts as a global interface to pip, meant to consume and install python packages that expose CLIs e.g. `poetry`. To install it, follow their instructions at [their documentation](https://pypa.github.io/pipx/installation/) to install `pipx`.
+
+### Using `poetry`
+
+Follow the instructions in [their documentation](https://python-poetry.org/docs/#installation) to install `poetry`. If you have `pipx` installed, you can install it with `pipx install poetry`. You can read more about this in their [installation documentation for `pipx`](https://python-poetry.org/docs/#installing-with-pipx).
```

Comment:
here maybe we could point the reader to the prefered way so we don't need to jump around their doc

`curl -sSL https://install.python-poetry.org | python3 -`

Reply:
thanks! I will add this.